src/deploy/functions/backend.ts

```diff
@@ -164,10 +164,18 @@ export interface TargetIds {
   project: string;
 }
 
+export interface SecretEnvVar {
+  key: string;
```

Comment:
We don't seem to use `key` at all, is this necessary?

Reply:
Being discussed on another thread.